contrib/node/src/python/pants/contrib/node/subsystems/resolvers/npm_resolver.py

```diff
@@ -14,6 +14,7 @@
 from pants.util.contextutil import pushd
```

Comment:
Good point.  I have created https://github.com/pantsbuild/pants/issues/5451 to track since we will need to touch a fairly sizable surface area to get all forms of "NpmResolve" out.